src/cmd/dc/delegate.rs

```diff
@@ -10,8 +10,8 @@ pub struct Cmd {
     /// Subdao to delegate DC to
     subdao: SubDao,
 
-    /// Router helium public key to delegate to
-    router: String,
+    /// Payer helium public key to delegate to
```

Comment:
```suggestion
    /// Public Helium payer key to delegate to
```
capitalizes Helium and pushes the words around.